pvlib/pvsystem.py

```diff
@@ -98,7 +98,8 @@ class PVSystem:
     arrays : iterable of Array, optional
         List of arrays that are part of the system. If not specified
         a single array is created from the other parameters (e.g.
-        `surface_tilt`, `surface_azimuth`). If `arrays` is specified
+        `surface_tilt`, `surface_azimuth`). Must contain at least one Array,
+        if empty a ValueError is raised. If `arrays` is specified
```

Comment:
Can a class docstring include a `Raises` section? If so, I'd add that section and describe the `ValueError` there also.

Does "if empty" imply length 0?  I would ignorantly interpret `arrays=None` as "empty".